### DISCUSSION TITLE
Clearing Logs

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5839] [Unsupported Versions] [answered]
Hello everyone,

I have read the documentation for how to clear both nsm and elastic data in security onion which seems pretty straightforward. Just curious if so-nsm-clear and so-elastic-clear will cause any adverse effects. The goal is to have a fresh grid with no previous logs from testing before placing into production. Any advice would be greatly appreciated!

Security Onion 2.3.80
Distributed GRID
1x Master
3x Search Nodes 
2x Sensor Nodes

Cole

**Comment by dougburks** [comment, marked as answer]
`so-nsm-clear` and `so-elastic-clear` are pretty simple. If you have any concerns, you can take a look at what they're doing under the hood:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-nsm-clear
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-elastic-clear
